chess-AI/streamlit_dashboard.py

```diff
@@ -55,18 +55,42 @@ def train_button(self):
     def load_files(self):
```

Comment:
https://pastebin.com/hsnU6seK

Reply:
Fixed the spacing pylint issues, other pylint issues that I am getting is stuff that was already on master